src/main/webapp/WEB-INF/jsp/simpleAttendanceSummary.jsp

```diff
@@ -95,6 +95,7 @@
 
         // Update the selected section information
         val = ${selectedSectionId};
+        document.getElementById('sectionId').value = val;
```

Comment:
looks like this page is using jquery, might want to use a jquery selector instead of using raw javascript to make it consistent. Maybe something like $("#sectionId").val(val);